cmd/jaeger/config-elasticsearch.yaml

```diff
@@ -17,6 +17,19 @@ extensions:
       some_storage:
         elasticsearch:
           index_prefix: "jaeger-main"
+#          indices:
+#            spans:
+#              date_layout: "2006-01-02"
```

Comment:
what happens to the index name pattern when hourly rollover is used? Just the date in the name is not granular enough.

Reply:
with following config:
```yaml
          indices:
            spans:
              date_layout: "2006-01-02-15"
              rollover_frequency: "hour"
```

the name of index in es will be `jaeger-main-jaeger-span-2024-08-02-02`